### ISSUE TITLE
Use helper function name instead of file name

### ISSUE THREAD

**Issue by nicolaisueper** [losttype/metalsmith-register-helpers#9]
Hey there,

I encountered a problem where handlebars could not find my helper. I debugged into the code and stumbled across [this line](https://github.com/losttype/metalsmith-register-helpers/blob/master/index.js#L26).

The problem was, that my helper function was named in camelCase but the containing file was named in kebab-case.

This leads to problems if you enforce kebab case file names inside your project.

Wouldn't it be better to do something like this: 

```switch (typeof helperContents) {
  case 'function':
    templateName = helperContents.name || file.split('.').shift();
```

So the helper would be named like the file name only if the function isn't named.

I'd work on it if I get your OK. :)

Greets
Nico

**Comment by kennethormandy** [comment]
Thanks for the detailed issue report. As long as the existing functionality works the same way, yeah, I’d accept a Pull Request :)

**Comment by kennethormandy** [comment]
Thanks for the Pull Request! Closed in #10.
